src/ierror.cpp

```diff
@@ -8,6 +8,7 @@ void usage( char *prog );
 
 int
 main( int argc, char **argv ) {
+    set_ips_display_name("ierror");
```

Comment:
Goes for all calls to `set_ips_display_name(...)` in this PR, do we prefer hard-coding the name or using another method to get the command name?

Reply:
Hard coding is fine. The names of these commands aren't going to change.